src/realtime_ui/trajectory.css

```
@@ -10,10 +10,14 @@ body { margin: 0; display: flex; overflow: hidden; background: var(--bg); color:
 button, input { font: inherit; }
 button { color: inherit; }
 .sidebar { flex: 0 0 248px; padding: 24px 18px; border-right: 1px solid var(--line); background: var(--sidebar); }
-.brand { display: flex; margin: 8px 7px 48px; align-items: center; gap: 9px; font-size: 24px; font-weight: 700; letter-spacing: -.04em; }
+.brand { display: flex; width: max-content; margin: 8px 7px 48px; padding: 0; align-items: center; gap: 9px; border: 0; background: transparent; font-size: 24px; font-weight: 700; letter-spacing: -.04em; cursor: pointer; }
 .brand img { width: 27px; height: 27px; object-fit: contain; }
-.session { width: 100%; height: 42px; margin-bottom: 38px; border: 1px solid #d9dce1; border-radius: 10px; background: #fff; font-size: 15px; cursor: pointer; }
-.sidebar-title { margin: 0 8px 22px; color: var(--muted); font-size: 15px; }
+.session { display: flex; width: 100%; height: 42px; margin-bottom: 38px; align-items: center; justify-content: center; gap: 8px; border: 1px solid #d9dce1; border-radius: 10px; background: #fff; font-size: 15px; cursor: pointer; }
+.session svg, .rail-action svg { width: 20px; height: 20px; fill: none; stroke: currentColor; stroke-width: 1.8; stroke-linecap: round; stroke-linejoin: round; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)